howso/engine/trainee.py

```diff
@@ -325,12 +333,12 @@ def features(self) -> SingleTableFeatureAttributes:
             return SingleTableFeatureAttributes({})
 
     @property
-    def metadata(self) -> Optional[Dict[str, Any]]:
+    def metadata(self) -> Dict[str, Any] | None:
```

Comment:
The doc string doesn't match the intent of the signature. Would this really ever return `None`, or would it really be an empty dict?